week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,48 +1,58 @@
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
-function mood() {
-  let isHappy = true;
+let isHappy = true;
 
-  if (isHappy) {
+function mood(isHappy) {
+
+  if (isHappy == true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
-function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
 
-  if (isBigEnough) {
+let num = 10;
+let isBigEnough = num;
+
+
+function greaterThan10(isBigEnough) {
+  
+  if (isBigEnough = num) {
     return "num is greater than or equal to 10";
   } else {
     return "num is not big enough";
   }
 }
 
+
+
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
-
+  let sortedLetters = letters.sort();
   return sortedLetters;
 }
 
+
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
-
+  let sliced = numbers.slice(0, 5);
   return sliced;
 }
 
+first5(1,2,3,4,5,6,7,8);
```

Comment:
The parameters in this function (e.g. (1,2,3,4,5,6,7,8)) won't do anything since there aren't any arguments inside the function on Line 38.